.metadata

```diff
@@ -0,0 +1,232 @@
+# Copyright 2019 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+# -------------------------------------------------------------------------
+
+# command line arguments
+params:
+  wso2is: wso2is-5.8.0
+  wso2is-analytics: wso2is-analytics-5.8.0
+  existing-version: 5.7.0
+  new-version: 5.8.0
+files:
+  # README.md
+  - file_path: README.md
+    relative_path: ~
+    file_type: txt
+    configurations:
+      - operation: replace
+        current_value: "v5.7.0.1"
+        new_value: "v5.8.0.1"
```

Comment:
Better parameterize this value so we can have uniformity with other vagrant resources